operators/serviceflow-operator/SerialServiceFlowOperator.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+
+const _ = require('lodash');
+const Promise = require('bluebird');
+const assert = require('assert');
+const fs = require('fs');
+const yaml = require('js-yaml');
+const path = require('path');
+const logger = require('../../common/logger');
+const errors = require('../../common/errors');
+const CONST = require('../../common/constants');
+const utils = require('../../common/utils');
+const apiServerClient = require('../../data-access-layer/eventmesh').apiServerClient;
+const BaseOperator = require('../BaseOperator');
+const TaskFabrik = require('./task/TaskFabrik');
+const FAILED = true;
```

Comment:
reason to name it in assertion is because, its a constant and if it were to be named as suggested then there is a possibility of it being false as well. In this case, I want to signify a boolean which has failed always.

Reply:
ok ,can leave it as it is.